src/yio-interface/integrationinterface.h

```diff
@@ -57,7 +57,7 @@ class IntegrationInterface {
     /**
      * @brief getAllAvailableEntities Can be implemented by integration
      */
-    virtual QStringList getAllAvailableEntities() = 0;
+    virtual QVariantList getAllAvailableEntities() = 0;
```

Comment:
Why is this change required?
What are the actual types that can be returned? E.g. does a list of integers make sense?